app/src/test/java/ar/scacchipa/jsonplaceholderviewer/data.kt

```diff
@@ -0,0 +1,78 @@
+package ar.scacchipa.jsonplaceholderviewer
+
+import ar.scacchipa.jsonplaceholderviewer.data.Comment
+import ar.scacchipa.jsonplaceholderviewer.data.Post
+
+fun getMockPosts() = listOf(
```

Comment:
I understand why you do this, you have like these utils func that are like global available so you can use it in different layers, for testing in the use case and also in the repositories, but have in mind you can do this because the same type (e.g Comment::class) is being used across all layers, but in the next projects that wont be like that any more, and there will be mappers between layers, so global funcionts like this wont be useful. Ideallly all these getStuffMethods() for mocks, should be like a private func inside the. mock class.